GsaGH/Components/5_Results/ReactionForceDiagrams.cs

```diff
@@ -0,0 +1,508 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Drawing;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using Grasshopper.Kernel;
+using Grasshopper.Kernel.Types;
+using GsaAPI;
+using GsaGH.Helpers.GH;
+using GsaGH.Parameters;
+using OasysGH;
+using OasysGH.Components;
+using OasysGH.Units;
+using OasysGH.Units.Helpers;
+using OasysUnits;
+using OasysUnits.Units;
+using Rhino.Geometry;
+
+namespace GsaGH.Components._5_Results
+{
+  /// <summary>
+  /// Component to display GSA node result contours
+  /// </summary>
+  public class ReactionForceDiagrams : GH_OasysDropDownComponent
+  {
+    #region core
+    protected override void RegisterInputParams(GH_InputParamManager pManager)
```

Comment:
Sorry, I meant putting those after the constructor (before all the other methods..)